Tests/Shared/IntegrationTest.swift

```diff
@@ -16,11 +17,11 @@ open class IntegrationTest: UnitTest {
     public private(set) var sampleDataFilesUtil: SampleDataFilesUtil!
 
     override open func setUp() {
-        setUp(modifySdkConfig: nil)
+        setUp(modifyModuleConfig: nil)
     }
 
-    open func setUp(modifySdkConfig: ((inout SdkConfig) -> Void)? = nil) {
-        super.setUp(modifySdkConfig: modifySdkConfig)
+    open func setUp(modifyModuleConfig: ((inout DataPipelineConfigOptions) -> Void)? = nil) {
```

Comment:
I deliberately removed it as our code is now mostly relying on params from `DataPipelineConfigOptions` instead of `SdkConfig`. So if any test is using `SdkConfig`, it should be updated to use `DataPipelineConfigOptions`. But so far it apparently hasn't broken anything, if it will, we should refactor the test to utilize `DataPipelineConfigOptions`.